projects/addon-editor/components/editor-socket/editor-socket.component.less

```diff
@@ -1,303 +1,10 @@
+@import './styles/base.less';
+@import './styles/text.less';
+@import './styles/link.less';
+@import './styles/list.less';
+@import './styles/media.less';
+@import './styles/placeholder.less';
+@import './styles/table.less';
 @import './styles/details.less';
 @import './styles/code-blocks.less';
```

Comment:
Can we write it like that?
```less
.tui-editor-socket {
  @import './styles/base.less';
  ...
}
```
It would be nice to have those separate styles independent of the container selector.